detection_rules/etc/packages.yaml

```diff
@@ -11,6 +11,8 @@ package:
       elastic:
         capabilities:
         - security
+        # Added siem category as these rules are used by the Elastic Security detection engine for security monitoring
```

Comment:
nit: not sure we need this comment.

Reply:
Keeping it very similar to what is in main of integrations as any deviation will cause an overwrite on the integration repo. Example - https://github.com/elastic/integrations/pull/14571/files#diff-0c931e1fccdd2b65f5fa4b9bf460f0e0cf94c311b9f3c95dfbdd60404d0e18b4